cloud_functions/ads_policy_monitor/init.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+bucketname='ads_policy_monitor'
+pr=$DEVSHELL_PROJECT_ID
+
+echo "Google Cloud Project selection..."
+until [[ "$yn" == [YyNn] ]]; do
+    proceed_with_project="You are about to install the Ads Policy Monitor into "
+    proceed_with_project+="the following project: $DEVSHELL_PROJECT_ID. Proceed? (y/n): "
+    read -p "$proceed_with_project" yn
+done
+
+if [[ "$yn" == "n"  || "$yn" == "N" ]]; then
+    read -p "In which project should the solution be installed on?:" pr
+    gcloud config set project "$pr"
+fi
+
+echo "Setting project_id in tfvars file..."
+echo 'project_id = "'"$pr"'"'  >> ./terraform/example.tfvars
```

Comment:
The `project_id` is already in `example.tfvars`, so this would add it twice.

You could either remove `project_id` from the file, or replace it with something like this:

```
sed -i '' "s/project_id = \'\'/project_id = \"my-project\"/" filename.txt
```